build-system/compile/compile.js

```diff
@@ -269,7 +269,7 @@ function generateCompilerOptions(outputDir, outputFilename, options) {
     // These arrays are filled in below.
     jscomp_error: [],
     jscomp_warning: [],
-    jscomp_off: [],
+    jscomp_off: ['uselessCode', 'suspiciousCode'],
```

Comment:
Move these settings down to the following section. This way, you can choose different settings for type checking and compilation.

https://github.com/ampproject/amphtml/blob/d64dfd84c58d4361061a35f71d66bd1a39f2ff7b/build-system/compile/compile.js#L292-L336

**Side note:** Maybe also worth updating the comment on line 269 to make this clearer.

Reply:
There's a pretty clear comment up there that I just entirely missed 🥇 . Done!